lib/browser/jasmine-patch.js

```diff
@@ -0,0 +1,3 @@
+var jasminePatch = require('../jasmine/patch');
+
+jasminePatch.apply();
```

Comment:
it's up to you. if we want the autoinstall behavior in cjs context then I'd
combine the two. if there are usecases when you wouldn't want autoapply
then keep it as is.

On Wed, May 13, 2015 at 12:34 AM Victor Berchet <notifications@github.com>
wrote:

> In lib/browser/jasmine-patch.js
> <https://github.com/angular/zone.js/pull/105#discussion_r30207901>:
>
> > @@ -0,0 +1,3 @@
> > +var jasminePatch = require('../jasmine/patch');
> > +
> > +jasminePatch.apply();
>
> The idea of this 'browser' folder is to generate IIFE in the dist/ folder.
>
> All other CJS module module in the lib/ folder are not immediately
> invoked, ie you can require them and invoke them when you want in a CJS
> context.
>
> Does this make sense or do you think that importing the module should also
> apply the patch ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/angular/zone.js/pull/105/files#r30207901>.
>